docs/customization.md

```diff
@@ -165,3 +165,19 @@ Content added to this file appears at the top of every page's main content betwe
 `_includes/nav_footer_custom.html`
 
 Any content added to this file will appear at the bottom left of the page below the site's navigation. By default an attribution to Just the Docs is displayed which reads, `This site uses Just the Docs, a documentation theme for Jekyll.`.
+
+### Custom Search Placeholder
+
+The placeholder text in the search bar (and its `aria-label`) is rendered by `_includes/search_placeholder_custom.html`, **after stripping HTML and whitespace**. By default, the content of the include is:
+
+```liquid
+Search {{site.title}}
+```
+
+Override this file to render a custom placeholder. For example,
+
+```liquid
+Find a post
+```
+
+Would make the placeholder text "Find a post". Any plaintext or liquid variable is supported.
```

Comment:
The first sentence is ungrammatical. Replace `Would` by `would` or `That would`.

Reply:
The `plaintext or` in that last sentence seems redundant and potentially confusing.

The phrase `liquid variable` might confuse readers who aren't familiar with Liquid. Perhaps add a link to somewhere that explains the use of Liquid in Jekyll themes, or drop the sentence. And (pedantically) it's Jekyll, rather than Liquid itself, that defines variables such as `site.title`.